examples/pyfunc/Pyfunc.ipynb

```diff
@@ -386,17 +388,53 @@
     "merlin.undeploy(v)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 4. Testing PyFunc Model Server Locally"
```

Comment:
the flow is bit odd, to do local after running dev, can this be mark as section 3 (optional) or have a short write up in section 3 to refer to section 4 for local run alternatively?

Reply:
I have put it as section 2.4 and mark it as optional.